### PR TITLE
Fixes #27003 - js linter nitpicks

### PR DESCRIPTION
**Pull request by ezr-ondrej**
New linter - new rules

### CODE REVIEW

File: webpack/assets/javascripts/react_app/common/I18n.js [hunk 1]
```diff
@@ -16,9 +16,7 @@ class IntlLoader {
     await this.fetchIntl();
     addLocaleData(
       await import(
-        /* webpackChunkName: 'react-intl/locale/[request]' */ `react-intl/locale-data/${
-          this.locale
-        }`
+        /* webpackChunkName: 'react-intl/locale/[request]' */ `react-intl/locale-data/${this.locale}`
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/common/I18n.js:19]: Any idea if the comment would do its job with a `\n` newline separating it from the string?

**Review by amirfefer — DISMISSED** [review]
Jest tests are green :+1: 
Thanks @ezr-ondrej  !

**Review by tbrisker — DISMISSED** [review]
TBH I'd be happy if there were more places where it didn't yell about new lines where it doesn't make sense just to satisfy a line length rule...
Just for context, what changed in the linter? do we need to bump a minimum version or something as well to make sure this doesn't get reverted when someone else edits these files?

**Comment by ezr-ondrej** [comment]
Yeah, well I like the linter to be strict and push me to keep the code looking better.
I do agree with the 80 characters line, but I am bad in keeping it by myself, but the linter's rule are not perfect and sometimes can get annoying, but I would say the benefit of similarly written code in whole codebase is much greater benefit, especially in pretty new code base, which has new devs "every day".

It had stoped forcing you to split the content of the templated strings now.
TBH I do not know where the linter got changed, but it would be nice to get it bit more stable for sure - maybe figure out our configs?

**Review by ohadlevy — COMMENTED** [review]
the change it self looks fine and I'm happy to merge, however it would be better if we figure out what exactly changed, so we can force the dev dependencies and not have someone with older versions revert those changes over time...?

**Comment by ezr-ondrej** [comment]
It was introduced by https://github.com/prettier/prettier/pull/5979 and released in version of `prettier 1.18` https://prettier.io/blog/2019/06/06/1.18.0.html
Pinning `"prettier": "^1.18.0"`.

**Review by tbrisker — APPROVED** [review]
Thanks @ezr-ondrej !

**Pull request merged**